fvm/meter/weighted_meter.go

```diff
@@ -418,18 +418,18 @@ func (m *WeightedMeter) MergeMeter(child *WeightedMeter) {
 	var childComputationUsed = child.computationUsed
 	m.computationUsed = m.computationUsed + childComputationUsed
 
-	for key, intensity := range child.ComputationIntensities() {
```

Comment:
this and the below `MemoryIntensties()` cannot be removed bcz they are used in https://github.com/onflow/flow-go/blob/master/fvm/transactionInvoker.go#L218